contextIO.go

```diff
@@ -95,6 +100,23 @@ func (c *ContextIO) NewRequest(method, q string, queryParams url.Values, body io
 	return req, nil
 }
 
+func (c *ContextIO) wait() {
+	if c.rate > 0 && c.lastRequest.Unix() > 0 {
+		sinceLast := time.Now().Sub(c.lastRequest)
+		sleep := c.rate - sinceLast
+		if sleep > 0 {
+			time.Sleep(sleep)
+		}
+	}
+	c.lastRequest = time.Now()
+}
+
+// SetRate the number of possible requests per minute
+func (c *ContextIO) SetRate(requestsPerMinute float64) {
```

Comment:
I didn't actually know the rate limits, I just noticed that I bumped into them, when making plentiful of requests. 

So the rate limit is actually dependant on what query, and to what account it is run? That would make it much more difficult to implement a sensible one in this package. I don't think I would have attempted it if I had that information. Isn't this something that the user should implement themselves?